packages/nextra-theme-docs/src/components/sidebar.tsx

```diff
@@ -28,16 +28,40 @@ import { Collapse } from './collapse'
 import { Anchor } from './anchor'
 import { DEFAULT_LOCALE } from '../constants'
 
-const TreeState: Record<string, boolean> = {}
-
-interface FolderProps {
-  item: PageItem | MenuItem | Item
-  anchors: string[]
-}
+const TreeState: Record<string, boolean> = Object.create(null)
 
 const Folder = memo(FolderImpl)
 
-function FolderImpl({ item, anchors }: FolderProps) {
+const classes = {
+  link: cn(
+    'flex rounded px-2 py-1.5 text-sm transition-colors [word-break:break-word]',
+    '[-webkit-tap-highlight-color:transparent] [-webkit-touch-callout:none] contrast-more:border'
+  ),
+  inactive: cn(
+    'hover:bg-gray-100 text-gray-500 hover:text-gray-900',
+    'dark:hover:bg-primary-100/5 dark:text-neutral-500 dark:hover:text-gray-50',
+    'contrast-more:text-gray-900 contrast-more:dark:text-gray-50',
+    'contrast-more:border-transparent contrast-more:hover:border-gray-900 contrast-more:dark:hover:border-gray-50'
+  ),
+  active: cn(
+    'bg-primary-50 text-primary-500 dark:bg-primary-500/10 font-bold',
+    'contrast-more:border-primary-500 contrast-more:dark:border-primary-500'
+  ),
+  list: 'flex gap-1 flex-col',
+  border: cn(
+    'relative before:absolute before:top-1.5 before:bottom-1.5',
+    'before:content-[""] before:w-px before:bg-gray-200 dark:before:bg-neutral-800',
+    'ltr:pl-3 rtl:pr-3 ltr:before:left-0 rtl:before:right-0'
+  )
```

Comment:
fixes missing borders 
<img width="300" alt="image" src="https://user-images.githubusercontent.com/7361780/188213523-8c9ca94e-b701-4b07-a439-759c12ed9c6f.png">

also previously border had a space in top and bottom, I made as before 
<img width="450" alt="image" src="https://user-images.githubusercontent.com/7361780/188213794-207200fd-bf0b-4db3-89ed-6aef91e7cbb3.png">